cms/jinja2/component_overrides/header/_macro.njk

```diff
@@ -176,8 +175,7 @@
                                 {% for item in params.navLinks.itemsList %}
                                     <li class="ons-grid__col ons-col-4@m ons-u-mb-no">
                                         {% for link in item.linksList %}
-                                            {# In the new header, nav items can have text and no link - later we would want to test that both are present before outputting a heading link #}
-                                            {% if link.text %}
+                                            {% if link.text and link.url %}
                                                 {% if link.url %}
```

Comment:
suggestion: this if/else and the one for the child could be dropped since the check is now for both text and url

Reply:
done.